.github/workflows/main.yml

```
@@ -47,4 +49,29 @@ jobs:
           group: Collaborators
           file: ApiDemos-debug.apk
           debug: true
-
+      - name: Upload apk artifact with mappings file
+        uses: ./
+        with:
+          appName: ${{ secrets.APP_NAME }}
+          token: ${{ secrets.API_TOKEN }}
+          group: Collaborators
+          file: ApiDemos-debug.apk
+          mappings: mappings.txt
+          releaseNotes: Has mappings
+          debug: true
+  ios:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get sample .ipa file for test purposes
+        run: wget https://github.com/bitbar/test-samples/blob/master/apps/ios/bitbar-ios-sample.ipa?raw=true -O sample-ios-app.ipa
+      - name: Upload ipa artifact with symbols file
+        uses: ./
+        with:
+          appName: ${{ secrets.IOS_APP_NAME }}
+          token: ${{ secrets.IOS_API_TOKEN }}
+          group: Collaborators
+          file: sample-ios-app.ipa
```

There should be something like:
```suggestion
          file: sample-ios-app.ipa
          symbols: path/to/crash_symbols.dSYM
```

but I wasn't able neither to find sample dSYM file nor generate one 🤷